app/dashboard/models.py

```diff
@@ -2608,6 +2608,19 @@ class Profile(SuperModel):
     automatic_backup = models.BooleanField(default=False, help_text=_('automatic backup profile to cloud storage such as 3Box if the flag is true'))
     as_representation = JSONField(default=dict, blank=True)
     tribe_priority = models.TextField(default='', blank=True, help_text=_('HTML rich description for what tribe priorities.'))
+
+    is_org = models.BooleanField(
```

Comment:
should we go in this way or just replicate what we have on data from github? https://api.github.com/users/gitcoinco
adding user_type= "Organization"  or "User"

Reply:
i figured a boolean was cheaper in terms of storage space.. plus theres only two possible user types